grafana-plugin/src/components/Policy/Policy.consts.ts

```diff
@@ -0,0 +1,42 @@
+import { SelectableValue } from '@grafana/data';
+
+const POLICY_DURATION_LIST: SelectableValue[] = [
```

Comment:
I think there also should be a 60 minutes options to match this with the existing backend options